src/term.rs

```diff
@@ -1180,10 +1179,11 @@ impl RichTerm {
                     pos,
                 )
             },
-            Term::Sealed(i, t) => {
-                let t = t.traverse(f, state, order)?;
+            Term::Sealed(i, t1, lbl) => {
+                let t1 = t1.traverse(f, state, order)?;
+                // Should we ingore the label? (i.e lbl)
```

Comment:
Yes, we should ignore the label. The label is just error reporting information. It is currently ignore while traversing a metavalue anyway, so let's be consistent here.

```suggestion
```